jobs/statsd/templates/config/config.js.erb

```diff
@@ -99,7 +99,7 @@ Optional Variables:
 
 */
 {
-  graphitePort: <%= p('carbon.cache.line_receiver_port') %>
+  graphitePort: <%= p('carbon_cache_line_receiver_port') %>
```

Comment:
In new BOSH namespacing doesn't make sense anymore as the properties are per job template.
I've added `statsd.` anyway as it looks more in line with the other properties.